py$(PYV): remove tox env folder $(@D)/.tox/py$(PYV)
py$(PYV): commands[0]> python -m pytest test_pytest_rerunfailures.py
============================= test session starts ==============================
platform sunos5 -- Python $(PYTHON_VERSION).X -- $(@D)/.tox/py$(PYV)/bin/python
cachedir: .tox/py$(PYV)/.pytest_cache
rootdir: $(@D)
collecting ... collected 103 items

test_pytest_rerunfailures.py::test_error_when_run_with_pdb PASSED
test_pytest_rerunfailures.py::test_execution_count_exposed PASSED
test_pytest_rerunfailures.py::test_extra_test_summary_for_reruns PASSED
test_pytest_rerunfailures.py::test_ini_file_parameters PASSED
test_pytest_rerunfailures.py::test_ini_file_parameters_override PASSED
test_pytest_rerunfailures.py::test_no_extra_test_summary_for_reruns_by_default PASSED
test_pytest_rerunfailures.py::test_no_rerun_on_class_setup_error_without_reruns PASSED
test_pytest_rerunfailures.py::test_no_rerun_on_pass PASSED
test_pytest_rerunfailures.py::test_no_rerun_on_skip_call PASSED
test_pytest_rerunfailures.py::test_no_rerun_on_skipif_mark PASSED
test_pytest_rerunfailures.py::test_no_rerun_on_strict_xfail_with_only_rerun_flag PASSED
test_pytest_rerunfailures.py::test_no_rerun_on_xfail_call PASSED
test_pytest_rerunfailures.py::test_no_rerun_on_xfail_mark PASSED
test_pytest_rerunfailures.py::test_no_rerun_on_xpass PASSED
test_pytest_rerunfailures.py::test_only_rerun_flag[only_rerun_texts0-True] PASSED
test_pytest_rerunfailures.py::test_only_rerun_flag[only_rerun_texts1-True] PASSED
test_pytest_rerunfailures.py::test_only_rerun_flag[only_rerun_texts10-True] PASSED
test_pytest_rerunfailures.py::test_only_rerun_flag[only_rerun_texts2-True] PASSED
test_pytest_rerunfailures.py::test_only_rerun_flag[only_rerun_texts3-False] PASSED
test_pytest_rerunfailures.py::test_only_rerun_flag[only_rerun_texts4-True] PASSED
test_pytest_rerunfailures.py::test_only_rerun_flag[only_rerun_texts5-True] PASSED
test_pytest_rerunfailures.py::test_only_rerun_flag[only_rerun_texts6-True] PASSED
test_pytest_rerunfailures.py::test_only_rerun_flag[only_rerun_texts7-True] PASSED
test_pytest_rerunfailures.py::test_only_rerun_flag[only_rerun_texts8-False] PASSED
test_pytest_rerunfailures.py::test_only_rerun_flag[only_rerun_texts9-False] PASSED
test_pytest_rerunfailures.py::test_only_rerun_flag_in_flaky_marker[AssertionError-None-True] PASSED
test_pytest_rerunfailures.py::test_only_rerun_flag_in_flaky_marker[AssertionError-ValueError-True] PASSED
test_pytest_rerunfailures.py::test_only_rerun_flag_in_flaky_marker[AssertionError: ERR-None-True] PASSED
test_pytest_rerunfailures.py::test_only_rerun_flag_in_flaky_marker[ValueError-AssertionError-False] PASSED
test_pytest_rerunfailures.py::test_only_rerun_flag_in_flaky_marker[ValueError-None-False] PASSED
test_pytest_rerunfailures.py::test_only_rerun_flag_in_flaky_marker[marker_only_rerun2-None-True] PASSED
test_pytest_rerunfailures.py::test_only_rerun_flag_in_flaky_marker[marker_only_rerun3-None-False] PASSED
test_pytest_rerunfailures.py::test_only_rerun_flag_in_flaky_marker[marker_only_rerun5-None-False] PASSED
test_pytest_rerunfailures.py::test_only_rerun_flag_in_flaky_marker[marker_only_rerun6-None-True] PASSED
test_pytest_rerunfailures.py::test_pytest_runtest_logfinish_is_called PASSED
test_pytest_rerunfailures.py::test_rerun_except_and_only_rerun[only_rerun_texts0-rerun_except_texts0-True] PASSED
test_pytest_rerunfailures.py::test_rerun_except_and_only_rerun[only_rerun_texts1-rerun_except_texts1-True] PASSED
test_pytest_rerunfailures.py::test_rerun_except_and_only_rerun[only_rerun_texts2-rerun_except_texts2-False] PASSED
test_pytest_rerunfailures.py::test_rerun_except_and_only_rerun[only_rerun_texts3-rerun_except_texts3-False] PASSED
test_pytest_rerunfailures.py::test_rerun_except_and_only_rerun[only_rerun_texts4-rerun_except_texts4-False] PASSED
test_pytest_rerunfailures.py::test_rerun_except_and_only_rerun[only_rerun_texts5-rerun_except_texts5-False] PASSED
test_pytest_rerunfailures.py::test_rerun_except_and_only_rerun[only_rerun_texts6-rerun_except_texts6-False] PASSED
test_pytest_rerunfailures.py::test_rerun_except_flag[rerun_except_texts0-True] PASSED
test_pytest_rerunfailures.py::test_rerun_except_flag[rerun_except_texts1-True] PASSED
test_pytest_rerunfailures.py::test_rerun_except_flag[rerun_except_texts2-True] PASSED
test_pytest_rerunfailures.py::test_rerun_except_flag[rerun_except_texts3-False] PASSED
test_pytest_rerunfailures.py::test_rerun_except_flag[rerun_except_texts4-True] PASSED
test_pytest_rerunfailures.py::test_rerun_except_flag[rerun_except_texts5-False] PASSED
test_pytest_rerunfailures.py::test_rerun_except_flag[rerun_except_texts6-True] PASSED
test_pytest_rerunfailures.py::test_rerun_except_flag[rerun_except_texts7-False] PASSED
test_pytest_rerunfailures.py::test_rerun_except_flag_in_flaky_marker[AssertionError-None-AssertionError-False] PASSED
test_pytest_rerunfailures.py::test_rerun_except_flag_in_flaky_marker[AssertionError-ValueError-AssertionError-False] PASSED
test_pytest_rerunfailures.py::test_rerun_except_flag_in_flaky_marker[AssertionError: ERR-None-AssertionError-False] PASSED
test_pytest_rerunfailures.py::test_rerun_except_flag_in_flaky_marker[CustomFailure-None-AssertionError-True] PASSED
test_pytest_rerunfailures.py::test_rerun_except_flag_in_flaky_marker[CustomFailure-None-CustomFailure-False] PASSED
test_pytest_rerunfailures.py::test_rerun_except_flag_in_flaky_marker[ValueError-AssertionError-AssertionError-True] PASSED
test_pytest_rerunfailures.py::test_rerun_except_flag_in_flaky_marker[ValueError-None-AssertionError-True] PASSED
test_pytest_rerunfailures.py::test_rerun_except_flag_in_flaky_marker[marker_rerun_except2-None-AssertionError-False] PASSED
test_pytest_rerunfailures.py::test_rerun_except_flag_in_flaky_marker[marker_rerun_except3-None-AssertionError-True] PASSED
test_pytest_rerunfailures.py::test_rerun_except_flag_in_flaky_marker[marker_rerun_except5-None-AssertionError-True] PASSED
test_pytest_rerunfailures.py::test_rerun_except_flag_in_flaky_marker[marker_rerun_except6-None-AssertionError-True] PASSED
test_pytest_rerunfailures.py::test_rerun_except_flag_in_flaky_marker[marker_rerun_except7-None-AssertionError-False] PASSED
test_pytest_rerunfailures.py::test_rerun_except_passes_setup_errors PASSED
test_pytest_rerunfailures.py::test_rerun_fails_after_consistent_setup_failure PASSED
test_pytest_rerunfailures.py::test_rerun_fails_after_consistent_test_failure PASSED
test_pytest_rerunfailures.py::test_rerun_on_class_scope_fixture_with_error_with_reruns PASSED
test_pytest_rerunfailures.py::test_rerun_on_class_setup_error_with_reruns PASSED
test_pytest_rerunfailures.py::test_rerun_on_module_fixture_with_reruns PASSED
test_pytest_rerunfailures.py::test_rerun_on_session_fixture_with_reruns PASSED
test_pytest_rerunfailures.py::test_rerun_on_setup_class_with_error_with_reruns PASSED
test_pytest_rerunfailures.py::test_rerun_passes_after_temporary_setup_failure PASSED
test_pytest_rerunfailures.py::test_rerun_passes_after_temporary_test_crash PASSED
test_pytest_rerunfailures.py::test_rerun_passes_after_temporary_test_failure PASSED
test_pytest_rerunfailures.py::test_rerun_passes_after_temporary_test_failure_with_flaky_mark PASSED
test_pytest_rerunfailures.py::test_rerun_report PASSED
test_pytest_rerunfailures.py::test_reruns_if_flaky_mark_is_called_with_positional_argument PASSED
test_pytest_rerunfailures.py::test_reruns_if_flaky_mark_is_called_without_options PASSED
test_pytest_rerunfailures.py::test_reruns_with_condition_marker[0-0] PASSED
test_pytest_rerunfailures.py::test_reruns_with_condition_marker[1-2] PASSED
test_pytest_rerunfailures.py::test_reruns_with_condition_marker[False-00] PASSED
test_pytest_rerunfailures.py::test_reruns_with_condition_marker[False-01] PASSED
test_pytest_rerunfailures.py::test_reruns_with_condition_marker[None-0] PASSED
test_pytest_rerunfailures.py::test_reruns_with_condition_marker[True-20] PASSED
test_pytest_rerunfailures.py::test_reruns_with_condition_marker[True-21] PASSED
test_pytest_rerunfailures.py::test_reruns_with_condition_marker[condition6-2] PASSED
test_pytest_rerunfailures.py::test_reruns_with_condition_marker[condition7-0] PASSED
test_pytest_rerunfailures.py::test_reruns_with_condition_marker[condition8-2] PASSED
test_pytest_rerunfailures.py::test_reruns_with_condition_marker[condition9-0] PASSED
test_pytest_rerunfailures.py::test_reruns_with_delay[-1] PASSED
test_pytest_rerunfailures.py::test_reruns_with_delay[0.0] PASSED
test_pytest_rerunfailures.py::test_reruns_with_delay[0] PASSED
test_pytest_rerunfailures.py::test_reruns_with_delay[1] PASSED
test_pytest_rerunfailures.py::test_reruns_with_delay[2.5] PASSED
test_pytest_rerunfailures.py::test_reruns_with_delay_marker[-1] PASSED
test_pytest_rerunfailures.py::test_reruns_with_delay_marker[0.0] PASSED
test_pytest_rerunfailures.py::test_reruns_with_delay_marker[0] PASSED
test_pytest_rerunfailures.py::test_reruns_with_delay_marker[1] PASSED
test_pytest_rerunfailures.py::test_reruns_with_delay_marker[2.5] PASSED
test_pytest_rerunfailures.py::test_reruns_with_string_condition[os.getpid() != -1-2] PASSED
test_pytest_rerunfailures.py::test_reruns_with_string_condition[sys.platform.startswith("non-exists") == False-2] PASSED
test_pytest_rerunfailures.py::test_reruns_with_string_condition_with_global_var PASSED
test_pytest_rerunfailures.py::test_run_session_teardown_once_after_reruns PASSED
test_pytest_rerunfailures.py::test_verbose PASSED

=============================== warnings summary ===============================
test_pytest_rerunfailures.py: 103 warnings
  $(PYTHON_DIR)/vendor-packages/pytest_reporter/__init__.py:1: DeprecationWarning: pkg_resources is deprecated as an API. See https://setuptools.pypa.io/en/latest/pkg_resources.html
    from pkg_resources import get_distribution, DistributionNotFound

test_pytest_rerunfailures.py: 206 warnings
  $(PYTHON_DIR)/vendor-packages/pkg_resources/__init__.py:2868: DeprecationWarning: Deprecated call to `pkg_resources.declare_namespace('paste')`.
  Implementing implicit namespace packages (as specified in PEP 420) is preferred to `pkg_resources.declare_namespace`. See https://setuptools.pypa.io/en/latest/references/keywords.html#keyword-namespace-packages
    declare_namespace(pkg)

test_pytest_rerunfailures.py: 103 warnings
  $(PYTHON_DIR)/vendor-packages/pkg_resources/__init__.py:2868: DeprecationWarning: Deprecated call to `pkg_resources.declare_namespace('flufl')`.
  Implementing implicit namespace packages (as specified in PEP 420) is preferred to `pkg_resources.declare_namespace`. See https://setuptools.pypa.io/en/latest/references/keywords.html#keyword-namespace-packages
    declare_namespace(pkg)

test_pytest_rerunfailures.py: 206 warnings
  $(PYTHON_DIR)/vendor-packages/pkg_resources/__init__.py:2868: DeprecationWarning: Deprecated call to `pkg_resources.declare_namespace('repoze')`.
  Implementing implicit namespace packages (as specified in PEP 420) is preferred to `pkg_resources.declare_namespace`. See https://setuptools.pypa.io/en/latest/references/keywords.html#keyword-namespace-packages
    declare_namespace(pkg)

test_pytest_rerunfailures.py: 103 warnings
  $(PYTHON_DIR)/vendor-packages/pkg_resources/__init__.py:2868: DeprecationWarning: Deprecated call to `pkg_resources.declare_namespace('repoze.sphinx')`.
  Implementing implicit namespace packages (as specified in PEP 420) is preferred to `pkg_resources.declare_namespace`. See https://setuptools.pypa.io/en/latest/references/keywords.html#keyword-namespace-packages
    declare_namespace(pkg)

test_pytest_rerunfailures.py: 103 warnings
  $(PYTHON_DIR)/vendor-packages/pkg_resources/__init__.py:2348: DeprecationWarning: Deprecated call to `pkg_resources.declare_namespace('repoze')`.
  Implementing implicit namespace packages (as specified in PEP 420) is preferred to `pkg_resources.declare_namespace`. See https://setuptools.pypa.io/en/latest/references/keywords.html#keyword-namespace-packages
    declare_namespace(parent)

test_pytest_rerunfailures.py: 103 warnings
  $(PYTHON_DIR)/vendor-packages/pkg_resources/__init__.py:2868: DeprecationWarning: Deprecated call to `pkg_resources.declare_namespace('sphinxcontrib')`.
  Implementing implicit namespace packages (as specified in PEP 420) is preferred to `pkg_resources.declare_namespace`. See https://setuptools.pypa.io/en/latest/references/keywords.html#keyword-namespace-packages
    declare_namespace(pkg)

test_pytest_rerunfailures.py: 103 warnings
  $(PYTHON_DIR)/vendor-packages/pkg_resources/__init__.py:2868: DeprecationWarning: Deprecated call to `pkg_resources.declare_namespace('zc')`.
  Implementing implicit namespace packages (as specified in PEP 420) is preferred to `pkg_resources.declare_namespace`. See https://setuptools.pypa.io/en/latest/references/keywords.html#keyword-namespace-packages
    declare_namespace(pkg)

test_pytest_rerunfailures.py: 1545 warnings
  $(PYTHON_DIR)/vendor-packages/pkg_resources/__init__.py:2868: DeprecationWarning: Deprecated call to `pkg_resources.declare_namespace('zope')`.
  Implementing implicit namespace packages (as specified in PEP 420) is preferred to `pkg_resources.declare_namespace`. See https://setuptools.pypa.io/en/latest/references/keywords.html#keyword-namespace-packages
    declare_namespace(pkg)

test_pytest_rerunfailures.py::test_rerun_passes_after_temporary_test_crash
  $(PYTHON_DIR)/vendor-packages/pytest_benchmark/logger.py:46: PytestBenchmarkWarning: Benchmarks are automatically disabled because xdist plugin is active.Benchmarks cannot be performed reliably in a parallelized environment.
    warner(PytestBenchmarkWarning(text))

-- Docs: https://docs.pytest.org/en/stable/how-to/capture-warnings.html
======== 103 passed, 2576 warnings ========
  py$(PYV): OK
  congratulations :)
